scripts/confirm_image.sh

```diff
@@ -12,33 +12,34 @@ if [[ "$2" != "--want-production" ]] && [[ "$2" != "--want-debug" ]]; then
     exit 2
 fi
 
```

Comment:
Could you manually validate this change is still correct? I spent a lot of time getting it wrong :D

Reply:
Yep! Made it `echo "Is production: ${is_production}"` and confirmed from the CI logs